src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/PresentationBuildTasks.csproj

```diff
@@ -279,6 +280,7 @@
     <Compile Include="MS\Internal\MarkupCompiler\CompilationUnit.cs" />
     <Compile Include="MS\Internal\MarkupCompiler\FileUnit.cs" />
     <Compile Include="MS\Internal\MarkupCompiler\MarkupCompiler.cs" />
+    <Compile Include="MS\Internal\MarkupCompiler\NETFrameworkGetRelativePath.cs" />
```

Comment:
Why not use `GetRelativePath.NETFramework` as a filename instead?

Reply:
Also consider PathInternal.cs.